### ISSUE TITLE
Dapr extension services block is not recognized

### ISSUE THREAD

**Issue by chan4lk** [dotnet/tye#1238]
### Describe the bug
I want to specify the metrics port for each service in dapr extension area. But when its included according to the schema it throws an exception.

### To Reproduce
1. Add following lines to the `tye.yaml`.
```yaml
extensions:
  - name: dapr
    components-path: ./components/tye
    config: config
    services:
     identityapi:
        metrics-port: 9093
```
2. Run command `tye run`
3. An Exception is thrown.
```shell
Loading Application Details...
Error parsing tye.yaml: (8, 5): Expected scalar value for key: "services".
```

### Further technical details
1. tye --version output
```
0.10.0-alpha.21420.1+7cec0655cc474aee8d3c3a3d180678985209d439
```
2. Debug output
```shell
Loading Application Details...
Error parsing tye.yaml: (8, 7): Expected scalar value for key: "services".
```
3. Platform - Windows

**Comment by philliphoff** [comment]
@chan4lk I apologize for the confusion; the per-service Dapr configuration properties are only available on the 0.11.x builds, not yet officially released ([but can be installed](https://github.com/dotnet/tye/blob/main/docs/getting_started.md#working-with-ci-builds), if desired).  At the moment, we don't have a great way to document the Tye configuration schema on a per-version basis (see #1204).

**Comment by philliphoff** [comment]
Closing in favor of #1204.
